pipelines/notify-slack-on-failure-sample/notify-slack-on-failure-sample.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: notify-slack-on-failure-sample
```

Comment:
I don't think `-sample` should be included here (applies to the file and dir names too). Yes, this is being added as a sample, but I expect (probably many) users to use it in production workflows